cmd/juju/crossmodel/remove.go

```diff
@@ -96,12 +121,25 @@ func (c *removeCommand) Run(ctx *cmd.Context) error {
 		}
 		c.offerSource = controllerName
 	}
+
+	if !c.assumeYes && c.force {
+		fmt.Fprintf(ctx.Stdout, removeOfferMsg, strings.Join(c.offers, ", "))
+
+		if err := jujucmd.UserConfirmYes(ctx); err != nil {
+			return errors.Annotate(err, "offer removal")
```

Comment:
Please have a test that checks this path. We have had in past output that was puzzling to users when something goes wrong.

Reply:
We don't test this path elsewhere as in general we really don't want to be overly prescriptive about user facing text. All the test would do is assert that the text in the output matches that in the code, not whether the text is any good or not.

Reply:
I don't think that we care much about exact wording here (although, we should somewhere!!!). But we do need to check what is being logged vs being presented to the user on stderr and stdout...

There have been cases of duplicate messages because we logged and returned and err (both go to stderr)

Reply:
I added an extra test